draft-ietf-quic-transport.md

```diff
@@ -542,68 +545,132 @@ the server supports, as shown below.
+
+After receiving a Server Stateless Retry packet, the next cryptographic
+handshake message MUST be sent by the client on stream 0 at an offset of 0.  The
+client discards any buffers it uses for generating retransmissions of handshake
+data and effectively starts the connection anew, keeping only the cryptographic
+handshake state.  The cryptographic handshake is continued, but the transport
+connection is reset.  Continuing the cryptographic handshake is necessary to
+ensure that an attacker cannot force a downgrade of any cryptographic
+parameters.
+
+The client constructs a new Client Cleartext packet
+({{packet-client-cleartext}}) for subsequent cryptographic handshake messages.
+
+The payload of this packet contains STREAM frames and could contain PADDING and
+ACK frames.
+
+
+### Client Cleartext Packet {#packet-client-cleartext}
+
+A Client Cleartext packet has a type octet of 0x84 and is sent when the client
+has received a packet from the server, which will be either a Server Cleartext
+packet or a Version Negotiation packet.
+
+The connection ID field in a Client Cleartext packet contains a connection ID
+that was selected by the server (see {{connection-id}}).
 
+The payload of this packet contains STREAM frames and could contain PADDING and
+ACK frames.  If this packet is sent in response to a Version Negotiation
+({{packet-version}}) or Server Stateless Retry packet
+({{packet-server-stateless}}), it MUST be padded to the minimum packet size for
+initial packets (see {{packetization}}).
 
-## Encrypted Packets {#encrypted-packet}
 
-Packets encrypted with either 0-RTT or 1-RTT keys may be sent with long headers.
-Different packet types explicitly indicate the encryption level for ease of
-decryption. These packets contain:
+### Server Cleartext Packet {#packet-server-cleartext}
 
-* Octet 0: 0x85, 0x86 or 0x87
-* Octets 1-8: Connection ID (initial or final)
-* Octets 9-12: Packet Number
-* Octets 13-16: Version
-* Octets 17+: Encrypted Payload
+A Server Cleartext packet has a type octet of 0x85.  It is used to carry
+acknowledgments and cryptographic handshake messages from the server.
 
-A first octet of 0x85 indicates a 0-RTT packet. After the 1-RTT keys are
-established, key phases are used by the QUIC packet protection to identify the
-correct packet protection keys. The initial key phase is 0. See {{QUIC-TLS}} for
-more details.
+The connection ID field in a Server Cleartext packet contains a connection ID
+that is chosen by the server (see {{connection-id}}).
+
+The payload of this packet contains STREAM frames and could contain PADDING and
+ACK frames.
+
+
+## Encrypted Packets {#packet-encrypted}
+
+Packets that are protected with 0-RTT packets are sent with long headers.
+Packets that are protected with 1-RTT keys MAY be sent with long headers.  The
+different packet types explicitly indicate the encryption level and therefore
+the keys that are used to remove packet protection.
```

Comment:
decryption is too imprecise unfortunately, this is the term we use in TLS because it is what an AEAD does